package.json

```
@@ -1,12 +1,15 @@
 {
-	"name": "@sapphire/template",
+	"name": "@sapphire/create-sapphire-bot",
 	"version": "0.0.1",
-	"description": "template",
+	"description": "A CLI utility to create your next Sapphire bot with a template.",
 	"main": "dist/index.js",
 	"types": "dist/index.d.ts",
 	"author": "@sapphire",
 	"license": "MIT",
 	"private": false,
+	"bin": {
+		"create-sapphire-bot": "./dist/cli.js"
+	},
```

```suggestion
	"bin": {
		"create-sapphire-bot": "./dist/cli.js"
		"csb": "./dist/cli.js"
	},
```

short aliases are always pog